server/opts.go

```diff
@@ -697,7 +697,21 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 			}
 			o.TrustedOperators = append(o.TrustedOperators, opc)
 		}
+
+		if o.AccountResolver == nil && len(o.TrustedOperators) == 1 {
```

Comment:
The call to `processConfigFileLine()` (this function) for a given field is random, in that the caller iterates through a map (which is random). So the question is: is it a misconfiguration to have a trusted operator's account server url and explicit "resolver" field? Because depending on the order, you may get "resolver" then "trusted op" (which in this case would ignore the trusted operator's account server url) or "trusted op" then "resolver" in which case we would replace it with the last one.

Reply:
Since we support MEM as well as URL I went with "resolver" takes precedence over AccountResolverUrl. (I'll add a comment as Wally suggested). 
This also seemed like a reasonable escape hatch in case you have a good reason to operate your own account resolver.
Do you think that's ok?

if we have "resolver" first this code will not trigger and we hold on to the value obtained from "resolver".

if we have "operator" first we use its account resolver's url. When "resolver" is specified as well, I will use whatever it has.

Reply:
Sounds good.